java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1698_java.html

```diff
@@ -17,10 +17,19 @@ <h2>Compliant Solution</h2>
 if (firstName != null &amp;&amp; firstName.equals(lastName)) { ... };
 </pre>
 <h2>Exceptions</h2>
-<p>Comparing two instances of the <code>Class</code> object will not raise an issue.</p>
+<p>Comparing two instances of the <code>Class</code> object will not raise an issue:</p>
 <pre>
 Class c;
-if(c == Integer.class) { //No issue raised
+if(c == Integer.class) { // No issue raised
+}
+</pre>
+<p>Comparing <code>Enum</code>s will not raise an issue:</p>
```

Comment:
is this exception is applied already?

Reply:
anyway this is unrelated to the commit, so I will revert it